tests/management/commands/test_create_command.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+import os
+import shutil
+
+from django.conf import settings
+from django.core.management import call_command
+from django.test import TestCase
+from six import StringIO
+
+try:
+    from unittest.mock import patch
+except ImportError:
+    from mock import patch
+
+
+class CreateCommandTests(TestCase):
+    """Tests for create_command command."""
+
+    def setUp(self):  # noqa
+        self.management_command_path = os.path.join(
+            settings.BASE_DIR, 'tests/testapp/management')
+        self.command_template_path = os.path.join(
+            settings.BASE_DIR, 'django_extensions/conf/command_template')
+
+        self.files = ['__init__.py', 'commands/__init__.py',
+                      'commands/sample.py']
```

Comment:
could you please either use a single line for this or use the following style:

```
self.files = [
    '__init__.py', 
    'commands/__init__.py',
    'commands/sample.py',
]
```

Same goes for the other lines in this file.
We allow lines to be pretty wide and don't hold to 80 chars wide :-)

Reply:
I like that style too, changed.